src/SIL.LCModel/Application/Impl/DomainDataByFlid.cs

```diff
@@ -42,7 +42,7 @@ internal sealed class DomainDataByFlid : ISilDataAccessManaged
 		/// Therefore, one should not use them for multi-session identity.
 		/// CmObject identity can only be guaranteed by using their Guids (or using '==' in code).
 		/// </remarks>
-		internal DomainDataByFlid(ICmObjectRepository cmObjectRepository, IStTextRepository stTxtRepository,
+		public DomainDataByFlid(ICmObjectRepository cmObjectRepository, IStTextRepository stTxtRepository,
```

Comment:
I'm not too keen on having these c'tors public even though it doesn't change anything in regards to the binary.

The reason is that all or at least most of these classes are implementation details and never intended to be created from outside of liblcm. Changing the c'tor to public muddles this intention a bit.